src/os_posix/os_fallocate.c

```diff
@@ -20,28 +20,29 @@
  */
 #if defined(HAVE_FALLOCATE) || (defined(__linux__) && defined(SYS_fallocate)) || \
   defined(HAVE_POSIX_FALLOCATE)
-#define WT_CALL_FUNCTION(op)                                         \
-    do {                                                             \
-        WT_DECL_RET;                                                 \
-        WT_FILE_HANDLE_POSIX *pfh;                                   \
-        bool remap;                                                  \
-                                                                     \
-        pfh = (WT_FILE_HANDLE_POSIX *)file_handle;                   \
-                                                                     \
-        remap = (offset != pfh->mmap_size);                          \
-        if (remap)                                                   \
-            __wt_prepare_remap_resize_file(file_handle, wt_session); \
-                                                                     \
-        WT_SYSCALL_RETRY(op, ret);                                   \
-        if (remap) {                                                 \
-            if (ret == 0)                                            \
-                __wt_remap_resize_file(file_handle, wt_session);     \
-            else {                                                   \
-                __wt_release_without_remap(file_handle);             \
-                WT_RET(ret);                                         \
-            }                                                        \
-        }                                                            \
-        return (0);                                                  \
+#define WT_CALL_FUNCTION(op)                                                   \
```

Comment:
Since we only use this macro as a wrapper for `fallocate` I think the name should reflect that. Maybe `WT_CALL_FALLOCATE()` or `WT_CALL_FALLOCATE_FUNC()`?

Reply:
I did not change this. Only the location of the backslash changed in my code.